evap/results/tests/test_views.py

```diff
@@ -471,20 +471,20 @@ def test_heading_question_filtering(self):
     def test_default_view_is_public(self):
         cache_results(self.evaluation)
 
-        # the view=public button should have class "active". The rest in-between is just noise.
-        expected_button = (
-            f'<a href="/results/semester/{self.evaluation.course.semester.pk}/evaluation/{self.evaluation.pk}'
-            + '?view=public" role="button" class="btn btn-sm btn-light active"'
-        )
-
         page_without_get_parameter = self.app.get(self.url, user=self.manager)
-        self.assertContains(page_without_get_parameter, expected_button)
+        self.assertTrue(page_without_get_parameter.context["view"] == "public")
```

Comment:
You can use
```suggestion
        self.assertEqual(page_without_get_parameter.context["view"], "public")
```
here and below, which will give a more descriptive error message if the assertion fails